### PR TITLE
Add OpenStack credentials request manifest

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

**Comment by Fedosin** [comment]
@adambkaplan done!

File: manifests/01-registry-credentials-request-openstack.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openstack-openshift-image-registry
```
Comment (dmage) [inline manifests/01-registry-credentials-request-openstack.yaml]:
```
azure-openshift-ingress            16m
cloud-credential-operator-iam-ro   17m
openshift-image-registry           17m
openshift-ingress                  17m
openshift-machine-api              17m
openshift-machine-api-azure        16m
```

@adambkaplan do we have a recommended pattern for these names?

Comment (adambkaplan) [reply manifests/01-registry-credentials-request-openstack.yaml]: looks like we don't have a uniform standard, but I'd stick with `openshift-image-registry-<platform>`

Comment (adambkaplan) [reply manifests/01-registry-credentials-request-openstack.yaml]: If we rename the original AWS request, hopefully the upgrade tests pass...

**Comment by adambkaplan** [comment]
Upgrade failure was infra-related

**Comment by dmage** [comment]
@Fedosin our CI stuck in some weird state, can you do `git commit --amend` and force push your changes?

**Pull request merged**